js/src/button.js

```diff
@@ -167,6 +170,33 @@ $(document)
     $(button).toggleClass(ClassName.FOCUS, /^focus(in)?$/.test(event.type))
   })
 
+$(window).on(Event.LOAD_DATA_API, () => {
+  // ensure correct active class is set to match the controls' actual values/states
+
+  // find all checkboxes/readio buttons inside data-toggle groups
+  let buttons = [].slice.call(document.querySelectorAll(Selector.DATA_TOGGLES_BUTTONS))
+  for (let i = 0, len = buttons.length; i < len; i++) {
+    const button = buttons[i]
+    const input = button.querySelector(Selector.INPUT)
+    if (input.checked || input.hasAttribute('checked')) {
+      $(button).addClass(ClassName.ACTIVE)
```

Comment:
button.classList.add(ClassName.ACTIVE) 😉

Reply:
sure, but i thought we're still jQuery-ing? didn't want to mix and match styles as it's used in the existing code. happy to change to pure DOM calls though

Reply:
we try to use jQuery only when it's needed (mostly for the event system)